Explorer/Assets/DCL/Infrastructure/Global/Dynamic/InitializationFlowContainer.cs

```diff
@@ -46,22 +48,26 @@ public static InitializationFlowContainer Create(
             var teleportStartupOperation = new TeleportStartupOperation(loadingStatus, realmContainer.RealmController, staticContainer.ExposedGlobalDataContainer.ExposedCameraData.CameraEntityProxy, realmContainer.TeleportController, staticContainer.ExposedGlobalDataContainer.CameraSamplingData, dynamicWorldParams.StartParcel);
 
             // TODO review why loadGlobalPxOperation is invoked on recovery
-            var loadGlobalPxOperation = new LoadGlobalPortableExperiencesStartupOperation(loadingStatus, selfProfile, staticContainer.FeatureFlagsCache, bootstrapContainer.DebugSettings, staticContainer.PortableExperiencesController);
+            var loadGlobalPxOperation = new LoadGlobalPortableExperiencesStartupOperation(loadingStatus, staticContainer.FeatureFlagsCache, bootstrapContainer.DebugSettings, staticContainer.PortableExperiencesController);
             var sentryDiagnostics = new SentryDiagnosticStartupOperation(realmContainer.RealmController, bootstrapContainer.DiagnosticsContainer);
 
+            var loadingOperations = new List<IStartupOperation>()
+            {
+                preloadProfileStartupOperation,
+                loadPlayerAvatarStartupOperation,
+                loadLandscapeStartupOperation,
+                checkOnboardingStartupOperation,
+                teleportStartupOperation,
+                ensureLivekitConnectionStartupOperation, // GateKeeperRoom is dependent on player position so it must be after teleport
+                sentryDiagnostics
+            };
+
+            if (!localSceneDevelopment)
+                loadingOperations.Add(loadGlobalPxOperation);
```

Comment:
Is not really clear why skip global portable experience on local scene development. Can you add a comment with the reason?

Reply:
Tackled at https://github.com/decentraland/unity-explorer/pull/3630/commits/f46cae078dff5f1374a83010d807572072dc690b